demo/app/ios/Runner/WalletInitiatedOpenID4CI.swift

```diff
@@ -25,12 +25,34 @@ public class WalletInitiatedOpenID4CI {
         
         let opts = Openid4ciNewInteractionOpts()
         opts!.add(trace!.traceHeader())
-        
+         
         self.walletInitiatedInteraction = Openid4ciNewWalletInitiatedInteraction(args, opts, nil)!
     }
     
     func getSupportedCredentials() throws -> Openid4ciSupportedCredentials{
-        let supportedCredentials = try walletInitiatedInteraction.supportedCredentials()
-        return supportedCredentials
+        return try walletInitiatedInteraction.supportedCredentials()
+    }
+    
+    func requestCredentialWithWalletInitiatedFlow(didVerificationMethod: ApiVerificationMethod, redirectURIWithParams: String) throws -> VerifiableCredential {
+        let credentials = try walletInitiatedInteraction.requestCredential(didVerificationMethod, redirectURIWithAuthCode: redirectURIWithParams, opts: nil)
+        return credentials.atIndex(0)!;
+    }
+    
+    func createAuthorizationURLWalletInitiatedFlow(scopes: ApiStringArray, credentialFormat: String, credentialTypes: ApiStringArray, clientID: String, redirectURI: String) throws -> String {
+        var createAuthURLError: NSError?
+        
+        let opts = Openid4ciNewCreateAuthorizationURLOpts()!.setScopes(scopes)
+        opts!.setIssuerState("https://api-gateway.dev.trustbloc.dev/issuer/bank_issuer/v1.0")
+        
+        let authorizationLink = walletInitiatedInteraction.createAuthorizationURL(clientID, redirectURI: redirectURI, credentialFormat: credentialFormat, credentialTypes: credentialTypes, opts: opts, error: &createAuthURLError)
+
+        return authorizationLink
+    }
+    
+    
+    func serializeDisplayData(issuerURI: String, vcCredentials: VerifiableCredentialsArray) -> String{
```

Comment:
`vc` already stands for `Verifiable Credential`, so if you were to "expand" our this variable name, it says "Verifiable Credential Credentials". I suggest renaming to `vcs` or `credentials`.